app/src/main/java/com/duckhang/caculator_demo/MainActivity.kt

```diff
@@ -1,11 +1,22 @@
 package com.duckhang.caculator_demo
 
-import androidx.appcompat.app.AppCompatActivity
 import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
 
 class MainActivity : AppCompatActivity() {
+
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
+        val fragment = Calculators()
+        val transaction = supportFragmentManager.beginTransaction()
+        transaction.add(R.id.fragment_caculator, fragment)
+        transaction.commit()
```

Comment:
```suggestion
        supportFragmentManager.beginTransaction()
                  .add(R.id.fragment_caculator, Calculators.newInstance())
                  .commit()
```
E xem convention mục 2.11